content/Reducing_Power_Consumption.md

```diff
@@ -5,18 +5,19 @@ batteries.
 # Using the HALT Instruction
 
 The HALT instruction should be used whenever possible to reduce power
-consumption & extend the life of the batteries. This command stops the
+consumption and extend battery life. This instruction halts the
 system clock, reducing the power consumption of both the CPU and ROM.
 
-The CPU will remain stopped until an interrupt occurs at which point the
-interrupt is serviced and then the instruction immediately following the
-HALT is executed.
+The CPU will remain halted until an interrupt occurs, at which point the
+interrupt is serviced (if IME is enabled and the interrupt is enabled in IE)
+and then execution continues at the instruction immediately following the
+HALT.
 
 Depending on how much CPU time is required by a game, the HALT
-instruction can extend battery life anywhere from 5 to 50% or possibly
+instruction can extend battery life anywhere from 5% to 50% or possibly
 more.
 
-When waiting for a vblank event, this would be a BAD example:
+When waiting for a v-blank event, this would be a BAD example:
 
 ```
  @@wait:
```

Comment:
Can you additionally edit this to RGBASM syntax?

Reply:
Sorry I'm not familiar with RGBASM syntax.